code/game/objects/structures/inflatable.dm

```diff
@@ -0,0 +1,328 @@
+/obj/item/inflatable
+	name = "inflatable wall"
+	desc = "A folded membrane which rapidly expands into a large cubical shape on activation."
+	icon = 'icons/obj/inflatable.dmi'
+	icon_state = "folded_wall"
+	w_class = 2
+
+/obj/item/inflatable/attack_self(mob/user)
+	playsound(loc, 'sound/items/zip.ogg', 75, 1)
+	user << "<span class='notice'>You inflate [src].</span>"
+	var/obj/structure/inflatable/R = new /obj/structure/inflatable(user.loc)
+	src.transfer_fingerprints_to(R)
+	R.add_fingerprint(user)
+	qdel(src)
+
+/obj/structure/inflatable
+	name = "inflatable wall"
+	desc = "An inflated membrane. Do not puncture."
+	density = 1
+	anchored = 1
+	opacity = 0
+
+	icon = 'icons/obj/inflatable.dmi'
+	icon_state = "wall"
+
+	var/health = 50.0
```

Comment:
why the .0, don't use decimals, it is useless and it confuses whoever newcoder goes to check this file